### PR TITLE
Feat/remove btn

### PR DESCRIPTION
**Pull request by Sharmyn28**
This PR has a suggestion for disabling the settings/configuration using a button.

Workflow:
- You find the button only in General section
- Clicking on the button will remove/delete the namespace and all the keys.
- After that will show you the first launch dialog.

### CODE REVIEW

File: src/components/dialog/dialog-first-launch.js [hunk 1]
```diff
@@ -6,66 +6,64 @@ import {
...
+                    </p>
 
-                        <strong>
-                            {i18n.t(
-                                'Any settings or configuration on a users device will be overwritten by settings applied here.'
-                            )}
-                        </strong>
+                    <strong>
+                        {i18n.t(
+                            'Any settings or configuration on a users device will be overwritten by settings applied here.'
```
Comment (jenniferarnesen) [inline src/components/dialog/dialog-first-launch.js:36]: user's

File: src/pages/layout.js [hunk 2]
```diff
@@ -96,48 +93,42 @@ class Layout extends React.Component {
...
                     <MainContent>
                         <Paper className={layoutStyles.paper__layout}>
                             <Switch>
-                                <Route
-                                    path="/"
-                                    exact
-                                    render={() => (
+                                <Route path="/" exact>
+                                    {this.state.openFirstLaunch === true ? (
+                                        <D2Shim>
```
Comment (jenniferarnesen) [inline src/pages/layout.js:118]: Is it possible to have the <D2Shim> component wrap the ternary statement all together?

```
<D2Shim>
{this.state.openFirstLaunch === true ? (...) : (...)}</D2Shim>

File: src/components/dialog/dialog-first-launch.js (hunk 1 above)
Comment (Sharmyn28) [reply src/components/dialog/dialog-first-launch.js:36]: thanks jennifer :)

File: src/pages/layout.js (hunk 2 above)
Comment (Sharmyn28) [reply src/pages/layout.js]: Just checked this option and it's working

**Pull request merged**